### DISCUSSION TITLE
Look into confidence ratings SDT models and cumulative logit link models

### DISCUSSION THREAD

**Discussion by venpopov** [venpopov/bmm#182] [Ideas]

**Comment by GidonFrischkorn** [comment]
I just stumbled across this article: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8419883/
And thought maybe it could be helpful for implementing the m-AFC SDT models assuming normal noise.

**Comment by GidonFrischkorn** [comment]
See progress on #108 (specifically the PRs linked to this issue)
